templates/frontend/objects/article_details.tpl

```diff
@@ -68,7 +68,7 @@
 				{* Published date *}
 				{if $article->getDatePublished()}
 					<div class="list-group-item date-published">
-						<strong>{translate key="submissions.published"}</strong>
+						<strong>{translate key="submissions.published"}:</strong>
```

Comment:
In your previous PR, you used the colon locale key: https://github.com/NateWr/bootstrap3/commit/b9e45d2e2edd1448ef6101e27b1815a4f4322015#diff-91eaa95f8172b06bd15727d9073c9b03R101

Can you do that here too? That way it can be modified in other languages as necessary.